src/app/post/[id]/components/postContent/style.css

```
@@ -0,0 +1,273 @@
+  align-items: center;
+  justify-content: space-between;
+  font-family: "Noto Sans KR", AppleGothic, Dotum, sans-serif;
+  cursor: pointer;
+}
+
+.button_div button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  height: 47px;
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)